src/pages/nft/class/_classId/index.vue

```diff
@@ -201,15 +201,15 @@ export default {
   },
   async mounted() {
     try {
-      const promises = [
-        this.updateDisplayNameList(this.iscnOwner),
-        this.updateNFTOwners(),
-        this.updateNFTHistory(),
-        this.getLIKEPrice(),
-        this.fetchISCNMetadata(),
-      ];
-      if (this.isWritingNFT) promises.push(this.updateNFTPurchaseInfo());
-      await Promise.all(promises);
+      this.updateDisplayNameList(this.iscnOwner);
+      this.updateNFTOwners();
+      this.updateNFTHistory();
+      this.getLIKEPrice();
+      const blockingPromises = [this.fetchISCNMetadata()];
+      if (this.isWritingNFT) {
+        blockingPromises.push(this.updateNFTPurchaseInfo());
+      }
+      await Promise.all(blockingPromises);
```

Comment:
Should we add catch for each of the promises if the error is not critical? If we use one catch or not awaiting, there might be unhandled promise that would kill the whole function